src/coreclr/src/jit/codegenarm64.cpp

```diff
@@ -5219,6 +5219,726 @@ void CodeGen::genArm64EmitterUnitTests()
 
 #endif // ALL_ARM64_EMITTER_UNIT_TESTS
 
+#ifdef ALL_ARM64_EMITTER_UNIT_TESTS
+    //
+    // Loads to /Stores from one, two, three, or four SIMD&FP registers
```

Comment:
"Loads to /Stores" => "Loads to and Stores from"? (same below)

I thought "/Stores" was a typo so I was confused reading it.